scripts/helper-update-version

```diff
@@ -0,0 +1,17 @@
+ #!/bin/bash
+ # Retrieve sources, get sha256sum, parse package version and update these strings in manifest, README and /conf/app.src
+ 
+url=$(curl -s https://api.github.com/repos/duniter/cesium/releases/latest | grep "browser_" | grep "web" | head -1 | cut -d\" -f4)
+wget -nc --quiet $url -P ./tmp
+CHECKSUM=$(sudo sha256sum ./tmp/cesium-*-web.zip | head -c 64)
+sed -i "s/SOURCE_SUM=.*/SOURCE_SUM=${CHECKSUM}/" ../conf/app.src
+
+PACKAGE_VERSION=$(curl -s https://api.github.com/repos/duniter/cesium/releases/latest | grep "tag_name" | head -1 | cut -d\" -f4)
+sed -i "s#SOURCE_URL=.*#SOURCE_URL=https://github.com/duniter/cesium/releases/download/${PACKAGE_VERSION}/cesium-${PACKAGE_VERSION}-web.zip#" ../conf/app.src
```

Comment:
You are already retrieving this URL upper.

May be you can extract the version from the URL.
I don’t know if it is easier.

Reply:
Done!